### ISSUE TITLE
Sound through 3.5mm headphone jack muffled/broken after wake from sleep

### ISSUE THREAD

**Issue by mankot14** [TheRandMan/Hackintosh---Dell-Chromebook-13-7310#19]
Have you noticed the sound through the headphone jack getting muffled/corrupted after waking the laptop from sleep? Can be remediated for a brief minute by going into Sound Preferences and switching the input back and forth from Line In to Internal. This doesn't stick and the issue comes back shortly after. A reboot manages to fixes it until the next time you wake from sleep.

**Comment by TheRandMan** [comment]
I don't think I have experienced that specific issue. I have noticed something similar though - with alcid=15 I would have speaker audio occasionally stop working until going into Sound Preferences. I've since switched back to alcid=3 since speaker sound works perfectly and I rarely use headphones.

I'm thinking it's an issue with AppleALC.kext

**Comment by mankot14** [comment]
Yeah I switched to layout 3 right after I upgraded OpenCore and that didn't seem to fix it either. It's probably just a sleep compatibility issue with AppleALC for this specific model. Google search didn't really turn up anything for any other notebooks either.
